docs/app_deploy.md

```diff
@@ -0,0 +1,56 @@
+# App Deploy
+
+## Deploy without heroku
+
+Setup your `master.key` that you used for your rails credentials file as `RAILS_MASTER_KEY`, domain as `HOST`, `RAILS_ENV` as `production` and `JEMALLOC_ENABLED` as `true`.
+Also setup any environment variables that need to be added to production after deploy depending on the environment of the server.
+Setup **Sidekiq**, **Redis**, **Postgres** and a **Task Scheduler** as we will need them to run our application.
+
+If using contenful for your blogs, setup the webhook to call your production server for clearing cache. `https://<HOST>/contentful/webhook` with the secret token (Header as `Authorization:Bearer`) that is set in the credentials file.
+
+## Heroku Setup
+
+### Automatic Deploy
+
+Deploy with an **app.json** file that helps Heroku pre-configure the application. You can click the button below to deploy using our template.
+
+[![Deploy](https://www.herokucdn.com/deploy/button.svg)](https://dashboard.heroku.com/new?template=https://github.com/danielpaul/RapidRails)
```

Comment:
this will deploy the rapid rails. instead show instructions on how they can deploy by connecting with GitHub. can link to their docs.